node/utils/simulation.ts

```diff
@@ -63,26 +65,30 @@ export const getSimulationPayloadsByItem = (
 
 export const orderFormItemToSeller = (
   orderFormItem: OrderFormItem & {
-    paymentData: any
+    paymentData: PaymentData
     ratesAndBenefitsData: RatesAndBenefitsData
     logisticsInfo: any[]
   }
 ) => {
   const unitMultiplier = orderFormItem.unitMultiplier ?? 1
+
   const [logisticsInfo] = orderFormItem.logisticsInfo
 
+  const sellingPrice = orderFormItem.priceDefinition.calculatedSellingPrice
+
+  const { price } = orderFormItem
+
+  const haveUnitMultiplier = unitMultiplier !== 1
+
+  const realPrice = haveUnitMultiplier
+    ? calculatePrice(unitMultiplier, sellingPrice, price)
+    : sellingPrice
+
   const commertialOffer = {
-    Price: orderFormItem.priceDefinition?.calculatedSellingPrice
-      ? Number(
-          (
-            orderFormItem.priceDefinition.calculatedSellingPrice /
-            (unitMultiplier * 100)
-          ).toFixed(2)
-        )
-      : orderFormItem.price / 100,
+    Price: Number((realPrice / 100).toFixed(3)),
```

Comment:
three decimal places is the minimum value we need to be able to guarantee the correct value when truncating the number. This does not generate failures in the behavior of the native IO components (I tested it on my side) and for Faststore it is ideal to generate the highest possible accuracy.

Reply:
I understand and agree that it shouldn't crash native IO components, I'm just concerned about headless stores that might expect this value to be to two decimal places

Reply:
I will mark this point as resolved since I understand that it would be a minor adjustment on the side of any headless store, but we can make an announcement before deploying the version.